cleaner.py

```diff
@@ -31,79 +28,73 @@ def chunks(l, n):
             yield l[i:i + n]
 
     @staticmethod
-    def get_all_dialogs():
+    def get_all_chats():
         dialogs = app.get_dialogs(pinned_only=True)
 
         dialog_chunk = app.get_dialogs()
         while len(dialog_chunk) > 0:
             dialogs.extend(dialog_chunk)
             dialog_chunk = app.get_dialogs(offset_date=dialogs[-1].top_message.date)
 
-        return dialogs
+        return [d.chat for d in dialogs]
 
-    def select_group(self):
-        dialogs = self.get_all_dialogs()
-        groups = [d for d in dialogs if d.chat.type in ('group', 'supergroup')]
+    def select_groups(self):
+        chats = self.get_all_chats()
+        groups = [c for c in chats if c.type in ('group', 'supergroup')]
 
+        print('Delete all your messages in')
         for i, group in enumerate(groups):
-            print(f'{i+1}. {group.chat.title}')
+            print(f'  {i+1}. {group.title}')
+        print(f'  {len(groups) + 1}. (!) ALL OF THE ABOVE (!)\n')
 
-        print('')
-
-        group_n = int(input('Insert group number: '))
-        if group_n not in range(1, len(groups)+1):
-            print('Invalid group number. Exiting...')
+        n = int(input('Insert option number: '))
+        if not 1 <= n <= len(groups) + 1:
+            print('Invalid option selected. Exiting...')
             exit(-1)
 
-        selected_group = groups[group_n - 1]
-
-        selected_group_peer = app.resolve_peer(selected_group.chat.id)
-        self.peer = selected_group_peer
-        self.chat_id = selected_group.chat.id
-
-        print(f'Selected {selected_group.chat.title}\n')
-
-        return selected_group, selected_group_peer
+        self.chats = groups if n == len(groups) + 1 else [groups[n - 1]]
+ 
+        print(f'\nSelected {", ".join(c.title for c in self.chats)}.\n')
```

Comment:
Reduce code complexity in two lines above, e.g. avoid compound statements and join inside f-string.